### DISCUSSION TITLE
How to change LED chain order ?

### DISCUSSION THREAD

**Discussion by GregS1000RR** [pixelit-project/PixelIt#160] [Q&A]
First of all, Congrats for this great project, very well documented.

I went trough the entire doc and web interface and NodeRED Matrix Config nodes, but didn't find how to solve it.

I put together 4x 8x8 WS2812 matrix LED I had at home, and could setup everything perfectly well, but...

Entire Text is MIRRORED. (My assumption is that the chineese guy who made this PCB layout made it with a top view, instead bottom, and the way the LED's are chained is reversed)

How can I tell your firmware to reverse the data order entering in the WS2812 daisy chain ?

Thank you again for this great project.

Regards
Gregory

Info :
ESP8266: WeMOS D1 Mini
WEB : OK
MQTT : OK
FW Ver 0.3.17 
NodeRED :  0.3.12

**Comment by o0shojo0o** [comment]
Settings for the arrangement of the LEDs can only be made using the predefined templates:
![image](https://user-images.githubusercontent.com/19333515/159161921-5a35af76-1339-4523-ad13-b5bfd82e4d2f.png)
Which again are stored here, unfortunately these can not be set at runtime, so that only a limited number are available, because the memory is running out. (https://learn.adafruit.com/adafruit-neopixel-uberguide/neomatrix-library)
https://github.com/o0shojo0o/PixelIt/blob/b89f238829265f2838df9beabc997bf365d19af4/src/PixelIt.ino#L2700-L2713

**Reply by o0shojo0o** [reply]
You can try what you need for a definition maybe we can include it.
